test/ttmlir/Silicon/TTMetal/simple_eltwise.mlir

```diff
@@ -1,5 +1,4 @@
 // RUN: ttmlir-opt --ttir-load-system-desc="path=%system_desc_path%" --ttir-to-ttmetal-backend-pipeline --ttmetal-serialize-to-binary="output=%t.ttm" %s | FileCheck %s
-// UNSUPPORTED: true
```

Comment:
I don't think this is safe, the test still segfaults right?

Reply:
yeah, we saw this segfault yesterday, and some folks reported it had been fixed. So we tried testing it out in CI but its still broken. Main has UNSUPPORTED: true for this test. CI for this PR also shows a failure: https://github.com/tenstorrent/tt-mlir/actions/runs/10600024736/job/29378026122?pr=513

I'll follow up on a separate issue